clusteradm/Makefile

```diff
@@ -44,3 +48,24 @@ recreate : delete
 
 	# create HPA for ngsa deployment for testing
 	@kubectl autoscale deployment ngsa --cpu-percent=40 --min=1 --max=2
+
+network-recreate: network-delete
```

Comment:
why add the network commands? We should be able to create on setup and never change

Reply:
I would remove for simplicity